rakelib/plugins-metadata.json

```diff
@@ -0,0 +1,905 @@
+{
+  "logstash-codec-cloudtrail": {
+    "default-plugins": false,
```

Comment:
@colinsurprenant @ph @purbon - it very much depends on whether the format is designed for humans first or the encapsulating class first.

For example (foolish but bear with me), we could simply have `"logstash-codec-dots" : {"id" : 101, (more properties)}` and some indexes ` "default-plugins" : [101, 104, 110]`
This would be friendly to the class but hard for humans.